codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/AddAwsConfigFields.java

```diff
@@ -199,6 +201,13 @@ public class AddAwsConfigFields implements GoIntegration {
                     .name(LOG_MODE_CONFIG_NAME)
                     .type(getAwsCoreSymbol("ClientLogMode"))
                     .documentation("Configures the events that will be sent to the configured logger.")
+                    .build(),
+            AwsConfigField.builder()
+                    .name(SDK_APP_ID)
+                    .type(getUniversalSymbol("string"))
+                    .documentation("The aws sdk app ID that could be retrieved from env var or shared config file" +
```

Comment:
This doesn't match the current generated setting:
> The sdk user agent app id to be added to http request's User-Agent header
	AppID string


Maybe need to regenerate?

**suggestion**: 
> The (optional) application specific identifier appended to the  User-Agent header.